README.md

```diff
@@ -1 +1,29 @@
-# ecommerce-team4
\ No newline at end of file
+# Ecommerce Project - Team4
+
```

Comment:
you can add the Figma link to our design write here @_@

Reply:
ok